pkg/kubelet/types/pod_update_test.go

```diff
@@ -171,6 +171,18 @@ func TestIsCriticalPod(t *testing.T) {
 			},
 			expected: true,
 		},
+		{
+			pod: v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "pod5",
+					Namespace: "kube-system",
+					Annotations: map[string]string{
+						"scheduler.alpha.kubernetes.io/critical-pod": "file",
```

Comment:
If you meant to add a static pod, the key should have been `ConfigSourceAnnotationKey`. What you have used is the deprecated critical annotation key.